freepbx/configure_users.php

```diff
@@ -28,13 +28,29 @@
 $results = $stmt->fetchAll(\PDO::FETCH_ASSOC);
 
 if ($_ENV['NETHVOICE_LDAP_SCHEMA'] === 'ad') {
+	# check if DOMAIN is in the right format
+	if (!preg_match('/DC=/i', $_ENV['NETHVOICE_LDAP_BASE'])) {
+		echo "Invalid DOMAIN format\n";
+		exit(1);
+	}
```

Comment:
Adding this check into a validation step of the `configure-module` action is more correct and allows to handle an error from the UI better.